uwblueprint/paramedics-react
# Sidebar cleanup for event and ccp dashboards

[Notion ticket](https://www.notion.so/uwblueprintexecs/SideBar-Clean-up-ab53c0748eec42859ab6214301f1da82)

[Figma link](https://www.figma.com/file/IpOPyv79gBwEohvOJvPV6f?embed_host=notion&kind=&node-id=3632%3A0&viewer=1)

## Changes
- Changed "View All Events" link, to "&larr; Home" link and moved it to top of sidebar
- Reduced spacing between icons and text in each row
- Increased spacing between sidebar rows
- Added blue highlight for directory, fixed highlight behaviour for consistency

## Screenshots
<img width="1437" alt="image" src="https://user-images.githubusercontent.com/38408276/102680531-0b3efd80-4187-11eb-9397-011634c02c12.png">

## Testing
- Navigate to CCP dashboard and Event dashboard
- Check that the changes appear on the sidebar
- Try clicking around making sure the original navigation behaviour was preserved
- Make sure highlighting works properly

## Checklist
>[Code review doc for reference](https://www.notion.so/uwblueprintexecs/Code-Review-a21fc85b00394f488e92d9d605f6b2bc)

before opening PR

filling out PR

after opening PR
